Update univ poly doc after automatic naming of universes. (#15981)
coq/coq: doc/sphinx/addendum/universe-polymorphism.rst

   Print selfpid.

Now :g:`pidentity` is used at two different levels: at the head of the
application it is instantiated at ``u`` while in the argument position
it is instantiated at ``u0``. This definition is only valid as long as
``u0`` is strictly smaller than ``u``, as shown by the constraints. Note
that if we made ``selfpid`` universe monomorphic, the
two universes (in this case ``u`` and ``u0``) would be declared in the global universe graph
with names ``selfpid.u0`` and ``selfpid.u1``. Since the constraints would also be global

> ``Print selfpid.`` would not show them and we would have to use :cmd:`Print Universes`.

Reviewer: Can you hard wrap these tighter?

   Print selfpid.

Now :g:`pidentity` is used at two different levels: at the head of the
application it is instantiated at ``u`` while in the argument position
it is instantiated at ``u0``. This definition is only valid as long as
``u0`` is strictly smaller than ``u``, as shown by the constraints.
Note that if we made ``selfpid`` universe monomorphic, the two
universes (in this case ``u`` and ``u0``) would be declared in the
global universe graph with names ``selfpid.u0`` and ``selfpid.u1``.

> Since the constraints would also be global ``Print selfpid.`` would
> not show them and we would have to use :cmd:`Print Universes`.

Reviewer 2: ```suggestion
Since the constraints would be global, ``Print selfpid.`` will
not show them, however they will be shown by :cmd:`Print Universes`.
```
